ui/components/component-library/text-field-base/text-field-base.scss

```diff
@@ -17,7 +17,8 @@
   border-color: var(--color-border-default);
 
   &--focused {
-    border-color: var(--color-primary-default);
+    outline: 5px auto Highlight; // firefox
+    outline: 5px auto -webkit-focus-ring-color; // chrome
```

Comment:
Removing border in favour of mimicking the browsers native focus styles which will be better for accessibility.

Reference: https://css-tricks.com/copy-the-browsers-native-focus-styles/

Reply:
Yessssss!